src/main/java/io/cdap/plugin/dlp/Utils.java

```diff
@@ -172,18 +173,34 @@ public static StructuredRecord getStructuredRecordFromTable(Table table, Structu
   }
 
   /**
-   * Creates a DLP Table from a StructuredRecord
+   * Creates a DLP Table using the all fields from StructuredRecord
```

Comment:
using all fields from the given StructuredRecord

Reply:
updated